Common/DataModel/TrackSelectionTables.h

```diff
@@ -19,21 +19,79 @@ namespace o2::aod
 namespace track
 {
 // Columns to store the DCA to the primary vertex
-DECLARE_SOA_COLUMN(DcaXY, dcaXY, float); //!
-DECLARE_SOA_COLUMN(DcaZ, dcaZ, float);   //!
+DECLARE_SOA_COLUMN(DcaXY, dcaXY, float); //! Impact parameter in XY of the track to the primary vertex
+DECLARE_SOA_COLUMN(DcaZ, dcaZ, float);   //! Impact parameter in Z of the track to the primary vertex
+
+struct TrackSelectionFlags {
+ public:
+  typedef uint16_t flagtype;
+  static constexpr flagtype kTrackType = 1 << 0;
+  static constexpr flagtype kPtRange = 1 << 1;
+  static constexpr flagtype kEtaRange = 1 << 2;
+  static constexpr flagtype kTPCNCls = 1 << 3;
+  static constexpr flagtype kTPCCrossedRows = 1 << 4;
+  static constexpr flagtype kTPCCrossedRowsOverNCls = 1 << 5;
+  static constexpr flagtype kTPCChi2NDF = 1 << 6;
+  static constexpr flagtype kTPCRefit = 1 << 7;
+  static constexpr flagtype kITSNCls = 1 << 8;
+  static constexpr flagtype kITSChi2NDF = 1 << 9;
+  static constexpr flagtype kITSRefit = 1 << 10;
+  static constexpr flagtype kITSHits = 1 << 11;
+  static constexpr flagtype kGoldenChi2 = 1 << 12;
+  static constexpr flagtype kDCAxy = 1 << 13;
+  static constexpr flagtype kDCAz = 1 << 14;
+  static constexpr flagtype kGlobalTrack = kTrackType | kPtRange | kEtaRange | kTPCNCls | kTPCCrossedRows | kTPCCrossedRowsOverNCls | kTPCChi2NDF | kTPCRefit | kITSNCls | kITSChi2NDF | kITSRefit | kITSHits | kGoldenChi2 | kDCAxy | kDCAz;
+};
 
 // Columns to store track filter decisions
-DECLARE_SOA_COLUMN(IsGlobalTrack, isGlobalTrack, uint8_t);       //!
-DECLARE_SOA_COLUMN(IsGlobalTrackSDD, isGlobalTrackSDD, uint8_t); //!
+DECLARE_SOA_COLUMN(IsGlobalTrack, isGlobalTrack, uint8_t);                     //!
+DECLARE_SOA_COLUMN(IsGlobalTrackSDD, isGlobalTrackSDD, uint8_t);               //!
+DECLARE_SOA_COLUMN(TrackCutFlag, trackCutFlag, TrackSelectionFlags::flagtype); //! Flag with the single cut passed flagged
```

Comment:
Isn't this duplicating information? If a certain set of those are 1, then isGlobalTrack is 1

Reply:
Yes, this is definitively duplicated information. I tried having an expression column (https://github.com/AliceO2Group/O2Physics/pull/691/commits/9ead612d125e0d1b3cf820b871a2b8ef9f3b2cc9) based on the flag content but this was not working out at runtime. I can try to reproduce the issue I was observing

Reply:
```
[17114:qa-event-track]: [12:14:10][STATE] Exiting FairMQ state machine
[17114:qa-event-track]: [12:14:10][FATAL] Unhandled o2::framework::runtime_error reached the top of main of o2-analysis-qa-event-track, device shutting down.
[17114:qa-event-track]:  Reason: Invalid combination of argument types 1 and 2
[17114:qa-event-track]:  Backtrace follow: 
[17114:qa-event-track]: 
```

This is why I reverted to having the duplicate information

Reply:
Some type cast seems to be missing. Can you send me the expression column in mattermost?

Reply:
But it is anyway in memory, so expression does not really help (it will use the same memory)...